lib/galaxy/tools/extract_dataset.xml

```diff
@@ -40,6 +40,22 @@
   </outputs>
   <help><![CDATA[
 
+========
+Synopsis
+========
+
+Extracts datasets from a collection based on either position or name.
```

Comment:
The name of the dataset is different than the element identifier. This should definitely be element identifier or identifier and not name.

Reply:
edited